configure checks whether the default port is available on the build system
disable the check as we don't want to end up with a build system dependant
configuration of the default port

diff -wpruN '--exclude=*.orig' a~/configure.ac a/configure.ac
--- a~/configure.ac	1970-01-01 00:00:00
+++ a/configure.ac	1970-01-01 00:00:00
@@ -253,14 +253,9 @@ AC_MSG_CHECKING(for default serial port
 
 if test "x$DFL_PORT" != x
 then
-	if test -c $DFL_PORT
-	then
 		AC_MSG_RESULT($DFL_PORT)
 		AC_DEFINE_UNQUOTED(DFL_PORT, "$DFL_PORT",
 		                   [Default /dev/... device])
-	else
-		DFL_PORT=""
-	fi
 fi
 
 if test "x$DFL_PORT" = x
